NewForce-at-Mountwest/bangazon-workforce-bucatini
# Tickets #2 & 4

Description
-Created the create employee view model.
-Created the edit employee view model.

Related Ticket(s)
1. Create New Employee
4. Edit Employee Information, Department, and Computer

Steps to Test
-To create a new Employee, navigate to ~/employee/create.
-Input employee's first and last name, assign a department, and supervisor checkbox (if you so desire)
-Click submit, data should post to database, and redirect will be broken, as "index" view has no been merged in yet.
-Can run "Select * From Employee" query on DB to check for new data.

To edit an Employee's details, navigate to ~/employee/edit/(employee's id #)' and you will be presented with the Employee's edit page.
-Change Employee's first name, last name, supervisory status, department and/or assigned computer.
-Submit should send updated employee information to Employee table
  (confirm with "SELECT FROM Employee WHERE Id = (edited employee's id)" )
-Submit will check if employee's assigned computer has changed, if so, a new record will be posted to ComputerEmployee table, and previously assigned computer will have the date/time noted in "Unassigned Date" field.
  (confirm with "SELECT FROM ComputerEmployee WHERE EmployeeId = (edited employee's id)", note that only one record should have an assign date and a null unassign date.


Checklist:
[ x ] I have commented my code, particularly in hard-to-understand areas
[ ] I have made corresponding changes to the documentation
[ ? ] My changes generate no new errors
